parm/config/gefs/config.resources

```diff
@@ -198,8 +198,8 @@ case ${step} in
         declare -x "wtime_${step}_gfs"="03:00:00"
         ;;
       "C384" | "C768" | "C1152")
-        declare -x "wtime_${step}"="06:00:00"
-        declare -x "wtime_${step}_gfs"="06:00:00"
+        declare -x "wtime_${step}"="14:00:00"
+        declare -x "wtime_${step}_gfs"="14:00:00"
```

Comment:
Ok. I'll revert the changes I made to config.resources. Instead of increasing `wtime`, it may be better to split the forecast job into segments to avoid timeouts. Fortunately, it looks like this is already marked as in issue (#2274)

Reply:
It may also be useful to allow for different wall times on different machines? However, I understand this may complicate the `config.resources` file.